stacker/commands/stacker/base.py

```diff
@@ -132,9 +133,6 @@ def get_context_kwargs(self, options, **kwargs):
         return {}
 
     def add_arguments(self, parser):
-        if not self.subcommands:
-            parser.set_defaults(run=self.run)
```

Comment:
no, we had stopped calling `add_arguments` on the base class, so this code was never getting called.